internal/engine/buildcontroller_test.go

```diff
@@ -102,3 +103,37 @@ func TestBuildControllerCrashRebuild(t *testing.T) {
 	assert.NoError(t, err)
 	f.assertAllBuildsConsumed()
 }
+
+func TestBuildControllerManualTrigger(t *testing.T) {
+	f := newTestFixture(t)
+	defer f.TearDown()
+
+	mount := model.Mount{LocalPath: "/go", ContainerPath: "/go"}
+	manifest := f.newManifest("fe", []model.Mount{mount})
+	f.Init(InitAction{
+		Manifests:   []model.Manifest{manifest},
+		WatchMounts: true,
+		TriggerMode: model.TriggerManual,
+	})
+
+	f.nextCall()
+	f.waitForCompletedBuildCount(1)
+
+	f.store.Dispatch(view.AppendToTriggerQueueAction{Name: "fe"})
```

Comment:
...wait, wouldn't this immediately be read off the triggerQueue and build? I'm confused. Why does this NOT cause a build but the one on ln. 135 does? or... because there are no pending file changes? if that's the case, add a comment / use `f.assertNoCall`?

Reply:
yep! because there are no pending changes

Reply:
done